### PR TITLE
AMS-5: Bug fixing to dns verification

### PR DESCRIPTION
**Pull request by MatiasMinchaca**
### Files changes
- #### docker-compose.yml is modified: 
  it adding the dns-verification service to build the container
- #### estadisticas.vue is modified in frontend/pages/_slug/verificaciones/dns/_id/ folder :
  add default value for dateFilteredStart and dateFilteredEnd
- #### package.json is modified in microservices/dns_validator/ folder:
  add script 'start', that runs the commands:
    - `yarn install` and `node index.js`
- #### nginx.conf is modified:
  - the server dns-verification:80 was add to the upstream
  - add the locations /dns with their respective servers

### Feature changes 
Dns verifications are already working and showing data
check it out on [http://localhost:3000/prueba/verificaciones/dns/](http://localhost:3000/prueba/verificaciones/dns)

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -8,18 +8,41 @@ http {
  
     upstream docker-nginx {
         server web-verification:80;
+        server dns-verification:80;
```
Comment (eddymdz) [inline nginx.conf:14]: @MatiasMinchaca this will not work

**Review by eddymdz — CHANGES_REQUESTED** [review]
@MatiasMinchaca check my comments

File: nginx.conf (hunk 1 above)
Comment (MatiasMinchaca) [reply nginx.conf:14]: I modified it putting another upstream

File: frontend/pages/_slug/verificaciones/dns/_id/estadisticas.vue [hunk 2]
```diff
@@ -102,7 +102,7 @@ export default {
         getData() {
             this.clearData();
 
-            return this.$axios.$get(`${this.getTenant.slug}/verifications/dns-verifications/${this.idParam}/get-data?start_date=${this.dateFilteredStart}&end_date=${this.dateFilteredEnd}`)
+            return this.$axios.$get(`${this.getTenant.slug}/verifications/dns-verifications/${this.idParam}/get-data?start_date=${this.dateFilteredStart || 0}&end_date=${this.dateFilteredEnd || 0}`)
```
Comment (eddymdz) [inline frontend/pages/_slug/verificaciones/dns/_id/estadisticas.vue:105]: @MatiasMinchaca why you did this change?

File: microservices/dns_validator/index.js [hunk 3]
```diff
@@ -50,6 +50,7 @@ app.post("/", async (req, res) => {
     console.log("error in dns test", e);
     response = e.response;
   } finally {
+    console.log(response)
```
Comment (eddymdz) [inline microservices/dns_validator/index.js]: @MatiasMinchaca remove extra code that is not related with the requirements

**Review by eddymdz — CHANGES_REQUESTED** [review]
@MatiasMinchaca please check my comments

File: frontend/pages/_slug/verificaciones/dns/_id/estadisticas.vue (hunk 2 above)
Comment (MatiasMinchaca) [reply frontend/pages/_slug/verificaciones/dns/_id/estadisticas.vue:105]: The default values in these variables are to avoid an error when making a request to the API since a null value is sent, and response error 500
